exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -66,7 +65,7 @@ func newCwLogsPusher(expConfig *Config, params component.ExporterCreateSettings)
 		return nil, err
 	}
 
-	expConfig.Validate()
+	_ = expConfig.Validate()
```

Comment:
So my suggestion was actually to do:
```suggestion
```

because the `Validate` method is already called [here](https://github.com/open-telemetry/opentelemetry-collector/blob/956db02db4fe81c310486f407417e28cb6fb337b/config/moved_config.go#L53)